docroot/themes/custom/mass_theme/mass_theme.theme

```diff
@@ -4451,6 +4492,15 @@ function mass_theme_preprocess_page(&$variables) {
     $allowed_regions = ['post_content'];
     mass_theme_add_regions_to_node($allowed_regions, $variables);
   }
+
+  // Add a main_class for the Data Listing views pages.
+  $data_listing_view_ids = [
+    'data_listing_all',
+    'data_listing_topic',
+  ];
+  if (isset($variables['page']['content']['system_main']['#view_id']) && in_array($variables['page']['content']['system_main']['#view_id'], $data_listing_view_ids)) {
+    $variables['main_class'] = 'data-listing-placeholder';
```

Comment:
@clairesunstudio This is where the placeholder class has been added for the data listing views pages.

Reply:
@clairesunstudio Did this class need to change? Is it being used?

Reply:
This is no longer used can you remove this? Sorry about it